riak/tests/test_all.py

```diff
@@ -1,442 +0,0 @@
-
-        results = client.bucket(self.bucket_name).multiget(keys)
-        for obj in results:
-            self.assertIsInstance(obj, RiakObject)
-            self.assertTrue(obj.exists)
-            if PY2:
-                self.assertEqual(obj.key, obj.encoded_data)
-            else:
-                self.assertEqual(obj.key, obj.data)
-
-    @unittest.skipIf(SKIP_POOL, 'SKIP_POOL is set')
-    def test_pool_close(self):
-        """
-        Iterate over the connection pool and close all connections.
-        """
-        # Do something to add to the connection pool
-        self.test_multiget_bucket()
-        if self.client.protocol == 'pbc':
-            self.assertGreater(len(self.client._pb_pool.resources), 1)
-        else:
-            self.assertGreater(len(self.client._http_pool.resources), 1)
-        # Now close them all up
-        self.client.close()
-        self.assertEqual(len(self.client._http_pool.resources), 0)
-        self.assertEqual(len(self.client._pb_pool.resources), 0)
-
-
-class RiakPbcTransportTestCase(BasicKVTests,
-                               KVFileTests,
-                               BucketPropsTest,
-                               TwoITests,
-                               LinkTests,
-                               ErlangMapReduceTests,
-                               JSMapReduceTests,
-                               MapReduceAliasTests,
-                               MapReduceStreamTests,
-                               EnableSearchTests,
-                               SearchTests,
-                               YZSearchTests,
-                               ClientTests,
-                               CounterTests,
-                               BucketTypeTests,
-                               SecurityTests,
-                               DatatypeIntegrationTests,
-                               BaseTestCase,
-                               unittest.TestCase,
-                               test_six.Comparison):
-
-    def setUp(self):
-        if not HAVE_PROTO:
-            self.skipTest('protobuf is unavailable')
-        self.host = PB_HOST
-        self.pb_port = PB_PORT
-        self.protocol = 'pbc'
-        super(RiakPbcTransportTestCase, self).setUp()
-
-    def test_uses_client_id_if_given(self):
-        zero_client_id = "\0\0\0\0"
-        c = self.create_client(client_id=zero_client_id)
-        self.assertEqual(zero_client_id, c.client_id)
-
-
-class RiakHttpTransportTestCase(BasicKVTests,
-                                KVFileTests,
-                                BucketPropsTest,
-                                TwoITests,
-                                LinkTests,
-                                ErlangMapReduceTests,
-                                JSMapReduceTests,
-                                MapReduceAliasTests,
-                                MapReduceStreamTests,
-                                EnableSearchTests,
-                                SolrSearchTests,
```

Comment:
Interesting, both pbc and http support the searches. Not sure why it was just in http.